docs/cloud-workload-security/README.md

```diff
@@ -125,7 +125,7 @@ Make an edit to any of the files mentioned above and then, from the root of the
 If a `*.go` file in `pkg/security` has been edited you will first need to generate the `*.json` files.
 Please run:
 ```sh
-go generate ./pkg/security/...
+inv -e security-agent.cws-go-generate
 # or only the specific file
 go generate ./path/to/the/touched/file
 ```
```

Comment:
v-- In the next line, we should also modify the
(at the root of `docs/cloud-workload-security`)
per:
(at the root of the repository)
Otherwize this leads to this error:
python3: can't open file './docs/cloud-workload-security/scripts/secl-doc-gen.py': [Errno 2] No such file or directory

Reply:
Oh ok, I see :)